plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageNetworkHandler.java

```diff
@@ -160,7 +160,10 @@ public void refreshConfiguration(@Nullable DBWNetworkProfile profile) {
                 configuratorEnableState = ControlEnableState.disable(configuratorPlaceholder);
             }
 
-            profileProvidedHint.setText(NLS.bind("Using configuration from profile ''<a href=\"#\">{0}</a>''", profile.getProfileName()));
+            profileProvidedHint.setText(NLS.bind(
+                "Using configuration from profile ''<a href=\"#\">{0}</a>''",
+                profile.getProfileName().trim()
```

Comment:
Shouldn't we also cut the profile name if it's too long?

Reply:
I don't think so, the UI adapts nicely anyway

Reply:
Again, you shouldn't trim it here; I believe you're the only one who has ever encountered it.